R/paths.R

```diff
@@ -1,5 +1,6 @@
 match_version <- function(version = "3") {
```

Comment:
Can we update the default version to `"5"` for all the methods? This implies that `{plumber}` will naturally update to v5.